src/map.config.json

```diff
@@ -58,7 +70,7 @@
       "format": "image/jpeg",
       "tileMatrixSetID": "EPSG:3857",
       "tileMatrixPrefix": "EPSG:3857:",
-      "activeOnStartup": false
+      "activeOnStartup": true
```

Comment:
CMIIW, this Rennes ortho is used for 3D map. Is it correct to activate it on startup (I thought the default is 2D map)

Reply:
i agree with Ismail.

Reply:
It's just more readable for now.
Anyway, the config and the styles are clearly work in progress, it's not a big deal ;)

Reply:
Not at all.